Update privacy-personal-data.md (#5396)
MicrosoftDocs/microsoft-365-docs: microsoft-365/managed-desktop/service-description/privacy-personal-data.md

ms.localizationpriority: normal
---

# Overview

Microsoft Managed Desktop is an IT-as-a-Service (ITaaS) service for enterprise cloud customers designed to keep employees’ Windows devices deployed and updated. It also provides IT service management and operations, monitors security and incident response, as well as providing user support. This documentation provides additional details on data platform and privacy compliance for Microsoft Managed Desktop.

## Microsoft Managed Desktop data sources and purpose

> Microsoft Managed Desktop provides its service to enterprise customers and properly administers customers’ enrolled devices by using data from various sources. These sources, including Azure Active Directory, Microsoft Intune, Microsoft Windows 10, and Microsoft Defender for Endpoint, provide a comprehensive view of the devices that Microsoft Managed Desktop manages. The service also uses these Microsoft services that Microsoft Managed Desktop to provide ITaaS capabilities:

Reviewer: ```suggestion
Microsoft Managed Desktop provides its service to enterprise customers and properly administers customers’ enrolled devices by using data from various sources. These sources, including Azure Active Directory, Microsoft Intune, Microsoft Windows 10, and Microsoft Defender for Endpoint, provide a comprehensive view of the devices that Microsoft Managed Desktop manages. The service also uses these Microsoft services to enable Microsoft Managed Desktop to provide ITaaS capabilities:
```

> - [Microsoft Windows 10 Enterprise] (https://docs.microsoft.com/en-us/windows/windows-10/) - for management of device setup experience, managing connections to other services, and operational support for IT pros.
> - [Windows Update for Business] (https://docs.microsoft.com/en-us/windows/deployment/update/waas-manage-updates-wufb) - uses Windows 10 Enterprise diagnostic data to provide additional information on Windows 10 update. 
> - [Microsoft Endpoint Manager] (https://docs.microsoft.com/en-us/mem/endpoint-manager-overview) – for device management and to keep your data secure.
>   - [Microsoft Azure Active Directory] (https://docs.microsoft.com/en-us/azure/active-directory/) - for authentication and identification of all user accounts. 
>   - [Microsoft Intune] (https://docs.microsoft.com/en-us/mem/intune/) – for distributing device configurations, device management and application management.
>   - [Endpoint Analytics] (https://docs.microsoft.com/en-us/mem/analytics/overview) – for analytical insights about device and app usage.
>   - [Windows Autopilot] (https://www.microsoft.com/en-us/microsoft-365/windows/windows-autopilot) – for device provisioning and deployment.
>   - [Microsoft Defender for Endpoint] (https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/) – provides security services such as device security monitoring and security intelligence data.
> - [Microsoft Managed Desktop] (https://endpoint.microsoft.com/#home)  – Data provided by the customer or generated by the service during running of the service.
> - [Microsoft 365 apps for enterprise] (https://www.microsoft.com/en-us/microsoft-365/enterprise/compare-office-365-plans?rtc=1) – for management of Microsoft 365 Apps.

Reviewer: ```suggestion
- [Microsoft Windows 10 Enterprise](/windows/windows-10/) - for management of device setup experience, managing connections to other services, and operational support for IT pros.
- [Windows Update for Business](/windows/deployment/update/waas-manage-updates-wufb) - uses Windows 10 Enterprise diagnostic data to provide additional information on Windows 10 update. 
- [Microsoft Endpoint Manager](/mem/endpoint-manager-overview) – for device management and to keep your data secure.
  - [Microsoft Azure Active Directory](/azure/active-directory/) - for authentication and identification of all user accounts. 
  - [Microsoft Intune](/mem/intune/) – for distributing device configurations, device management and application management.
  - [Endpoint Analytics](/mem/analytics/overview) – for analytical insights about device and app usage.
  - [Windows Autopilot](/microsoft-365/windows/windows-autopilot) – for device provisioning and deployment.
  - [Microsoft Defender for Endpoint](/microsoft-365/security/defender-endpoint/) – provides security services such as device security monitoring and security intelligence data.
- [Microsoft Managed Desktop](https://endpoint.microsoft.com/#home)  – Data provided by the customer or generated by the service during running of the service.
- [Microsoft 365 apps for enterprise](/microsoft-365/enterprise/compare-office-365-plans?rtc=1) – for management of Microsoft 365 Apps.
```

## Microsoft Managed Desktop data process and storage

> Microsoft Managed Desktop relies on data from multiple Microsoft products and services to provide its service to enterprise customers. To accomplish the goal of protecting and maintaining enrolled devices, we process and copy data from these services to Microsoft Managed Desktop. When we process data, we follow the documented directions you provide, as referenced in the Online Services Terms and Microsoft Privacy Statement. When we process data, we follow the documented directions you provide, as referenced in the [Online Services Terms] (https://www.microsoft.com/en-us/licensing/product-licensing/products) and [Microsoft Privacy Statement] (https://privacy.microsoft.com/en-us/privacystatement). Microsoft Managed Desktop’s processor duties include ensuring appropriate confidentiality, security, and resilience. Microsoft Managed Desktop employs additional privacy and security measures to ensure proper handling of personal identifiable data. 

Reviewer: ```suggestion
Microsoft Managed Desktop relies on data from multiple Microsoft products and services to provide its service to enterprise customers. To accomplish the goal of protecting and maintaining enrolled devices, we process and copy data from these services to Microsoft Managed Desktop. When we process data, we follow the documented directions you provide, as referenced in the Online Services Terms and Microsoft Privacy Statement. When we process data, we follow the documented directions you provide, as referenced in the [Online Services Terms](https://www.microsoft.com/licensing/product-licensing/products) and [Microsoft Privacy Statement](https://privacy.microsoft.com/privacystatement). Microsoft Managed Desktop’s processor duties include ensuring appropriate confidentiality, security, and resilience. Microsoft Managed Desktop employs additional privacy and security measures to ensure proper handling of personal identifiable data. 
```

## Microsoft Managed Desktop data storage and staff location

> Microsoft Managed Desktop stores its data in the Azure data centers in the United States. Personal data obtained by Microsoft Managed Desktop and other services is required to keep the service operational. If device is removed from Microsoft Managed Desktop, we keep personal data for a maximum of 30 days except for alert data collected by Microsoft Defender for Endpoint, which is stored for 180 days for security purposes. For more information on data retention, see [Data retention, deletion, and destruction in Microsoft 365 - Microsoft Service Assurance | Microsoft Docs] (https://docs.microsoft.com/compliance/assurance/assurance-data-retention-deletion-and-destruction-overview).

Reviewer: ```suggestion
Microsoft Managed Desktop stores its data in the Azure data centers in the United States. Personal data obtained by Microsoft Managed Desktop and other services are required to keep the service operational. If a device is removed from Microsoft Managed Desktop, we keep personal data for a maximum of 30 days except for alert data collected by Microsoft Defender for Endpoint, which is stored for 180 days for security purposes. For more information on data retention, see [Data retention, deletion, and destruction in Microsoft 365](/compliance/assurance/assurance-data-retention-deletion-and-destruction-overview).
```

Microsoft Managed Desktop Engineering Operations and Security Operations teams are located in the United States and India. 

## Microsoft Windows 10 diagnostic data

> Microsoft Managed Desktop uses [Windows 10 Enhanced diagnostic data*] (https://docs.microsoft.com/windows/privacy/windows-diagnostic-data) to keep Windows secure, up to date, troubleshoot problems, and make product improvements. The enhanced diagnostic data setting includes more detailed information about the devices enrolled in Microsoft Managed Desktop and their settings, capabilities, and device health. When enhanced diagnostic data is selected, data including required diagnostic data are collected. See Changes to Windows diagnostic data collection - Windows Privacy | Microsoft Docs for more information about the Windows 10 diagnostic data setting and data collection.

Reviewer: ```suggestion
Microsoft Managed Desktop uses [Windows 10 Enhanced diagnostic data*](/windows/privacy/windows-diagnostic-data) to keep Windows secure, up to date, troubleshoot problems, and make product improvements. The enhanced diagnostic data setting includes more detailed information about the devices enrolled in Microsoft Managed Desktop and their settings, capabilities, and device health. When enhanced diagnostic data is selected, data, including required diagnostic data, are collected. See [Changes to Windows diagnostic data collection](/windows/privacy/changes-to-windows-diagnostic-data-collection) for more information about the Windows 10 diagnostic data setting and data collection.
```

> *Note that in the future version of Windows, the telemetry taxonomy will change.  Microsoft Managed Desktop is committed to processing only the data that the service needs.  While this will mean the diagnostic level will change to Optional, Microsoft Managed Desktop will implement the limited diagnostic policies to fine tune diagnostic data collection required for the service.  More details, see [Changes to Windows diagnostic data collection - Windows Privacy | Microsoft Docs] (https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.microsoft.com%2Fen-us%2Fwindows%2Fprivacy%2Fchanges-to-windows-diagnostic-data-collection&data=04%7C01%7CSaki.Tong%40microsoft.com%7C5e8cc0a1356b43b18a3308d92c4b375e%7C72f988bf86f141af91ab2d7cd011db47%7C0%7C0%7C637589523823867011%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=LD4FD%2BWEQ13%2BU1eCBLtKZiy8HZLgqCnZWldaNhs%2FeaM%3D&reserved=0) 

Reviewer: ```suggestion
*Note that in future versions of Windows, the telemetry taxonomy will change. Microsoft Managed Desktop is committed to processing only the data that the service needs. While this will mean the diagnostic level will change to Optional, Microsoft Managed Desktop will implement the limited diagnostic policies to fine-tune diagnostic data collection required for the service. For more details, see [Changes to Windows diagnostic data collection](/windows/privacy/changes-to-windows-diagnostic-data-collection).
```

Microsoft Managed Desktop only processes and stores system-level data from Windows 10 optional diagnostic data originating from enrolled devices such as application and device reliability and performance information. Microsoft Managed Desktop does not process and store customers’ personal data such as chat and browser history, voice, text, or speech data. 

> For more information about the diagnostic data collection of Microsoft Windows 10, see the [Where we store and process personal data] (https://privacy.microsoft.com/privacystatement#mainwherewestoreandprocessdatamodule) section of the [Microsoft Privacy Statement] (https://privacy.microsoft.com/privacystatement).

Reviewer: ```suggestion
For more information about the diagnostic data collection of Microsoft Windows 10, see the [Where we store and process personal data](https://privacy.microsoft.com/privacystatement#mainwherewestoreandprocessdatamodule) section of the Microsoft Privacy Statement.
```

## Microsoft Windows Update for Business
Microsoft Windows Update for Business uses data from Windows diagnostics to analyze update status and failures. Microsoft Managed Desktop leverages this data and uses it to mitigate and resolve problems to ensure that all registered devices are up to date based on a predefined update cadence.

## Microsoft Azure Active Directory

> Identify data used by Microsoft Managed Desktop is stored by Azure Active Directory (AAD) in a geographical location based on the location provided by the organization when subscribing to Microsoft online services such as Microsoft Apps for enterprise and Azure. Identify data used by Microsoft Managed Desktop is stored by Azure Active Directory (AAD) in a geographical location based on the location provided by the organization when subscribing to Microsoft online services such as Microsoft Apps for enterprise and Azure. For more information on where your AAD data is located, see [Azure Active Directory- Where is your data located?] (https://msit.powerbi.com/view?r=eyJrIjoiODdjOWViZDctMWRhZS00ODUzLWI4MmQtNWM5NjBkZTBkNjFlIiwidCI6IjcyZjk4OGJmLTg2ZjEtNDFhZi05MWFiLTJkN2NkMDExZGI0NyIsImMiOjV9)

Reviewer: ```suggestion
Identifying data used by Microsoft Managed Desktop is stored by Azure Active Directory (Azure AD) in a geographical location based on the location provided by the organization when subscribing to Microsoft online services, such as Microsoft Apps for enterprise and Azure. Identifying data used by Microsoft Managed Desktop is stored by Azure AD in a geographical location based on the location provided by the organization when subscribing to Microsoft online services such as Microsoft Apps for enterprise and Azure. For more information on where your Azure AD data is located, see [Azure Active Directory - Where is your data located?](https://msit.powerbi.com/view?r=eyJrIjoiODdjOWViZDctMWRhZS00ODUzLWI4MmQtNWM5NjBkZTBkNjFlIiwidCI6IjcyZjk4OGJmLTg2ZjEtNDFhZi05MWFiLTJkN2NkMDExZGI0NyIsImMiOjV9)
```

## Microsoft Intune

> Microsoft Intune collects, processes, and shares data to Microsoft Managed Desktop to support business operations and the service. See [Data collection in Intune - Microsoft Intune | Microsoft Docs] (https://docs.microsoft.com/mem/intune/protect/privacy-data-collect) for more information about the data collected in Intune. 

Reviewer: ```suggestion
Microsoft Intune collects, processes, and shares data to Microsoft Managed Desktop to support business operations and services. See [Data collection in Intune](/mem/intune/protect/privacy-data-collect) for more information about the data collected in Intune. 
```

> For more information on where your customer data is stored for Microsoft Intune, see [Microsoft 365 data locations - Microsoft 365 Enterprise | Microsoft Docs] (https://docs.microsoft.com/microsoft-365/enterprise/o365-data-locations?view=o365-worldwide). Intune respects the storage location elections made by the administrator for customer data.

Reviewer: ```suggestion
For more information on Microsoft Intune data locations, see [Where your Microsoft 365 customer data is stored](/microsoft-365/enterprise/o365-data-locations?view=o365-worldwide). Intune respects the storage location selections made by the administrator for customer data.
```

## Microsoft Defender for Endpoint

> Microsoft Defender for Endpoint collects and stores information for devices enrolled in Microsoft Managed Desktop for administration, tracking, and reporting purposes. Information collected includes file data (such as file names, size, and hashes), process data (running processes, hashes), registry data, network connection data, and device details (such as device identifiers, device names, and the operating system version). See [Microsoft Defender for Endpoint data storage and privacy | Microsoft Docs] (https://docs.microsoft.com/microsoft-365/security/defender-endpoint/data-storage-privacy?view=o365-worldwide#what-data-does-microsoft-defender-atp-collect) for more information on Microsoft Defender for Endpoint’s data collection and storage location. 

Reviewer: ```suggestion
Microsoft Defender for Endpoint collects and stores information for devices enrolled in Microsoft Managed Desktop for administration, tracking, and reporting purposes. Information collected includes file data (such as file names, size, and hashes), process data (running processes, hashes), registry data, network connection data, and device details (such as device identifiers, device names, and the operating system version). See [Microsoft Defender for Endpoint data storage and privacy](/microsoft-365/security/defender-endpoint/data-storage-privacy?view=o365-worldwide#what-data-does-microsoft-defender-atp-collect) for more information on Microsoft Defender for Endpoint’s data collection and storage locations. 
```

## Microsoft 365 Apps for enterprise 

> Microsoft 365 Apps for enterprise collects and shares data with Microsoft Managed Desktop to ensure those apps are up to date with the latest version based on predefined update channels managed by Microsoft Managed Desktop.  See [Microsoft Defender for Endpoint data storage and privacy | Microsoft Docs] (https://docs.microsoft.com/microsoft-365/security/defender-endpoint/data-storage-privacy?view=o365-worldwide#what-data-does-microsoft-defender-atp-collect) for more information on M365 Apps’ data collection and storage location.

Reviewer: ```suggestion
Microsoft 365 Apps for enterprise collects and shares data with Microsoft Managed Desktop to ensure those apps are up to date with the latest version based on predefined update channels managed by Microsoft Managed Desktop.  See [Microsoft Defender for Endpoint data storage and privacy](/microsoft-365/security/defender-endpoint/data-storage-privacy?view=o365-worldwide#what-data-does-microsoft-defender-atp-collect) for more information on Microsoft 365 Apps's data collection and storage locations.
```

## Major data change notification

> Microsoft Managed Desktop follows a change control process as outlined in our service communication framework. Customers are notified through the M365 Message Center and Microsoft Managed Desktop Portal of both security incidents and major changes to the service. Changes to the types of data gathered and where it is stored is considered a material change. We will provide a minimum of 30 days advanced notification of this change as is standard practice for Microsoft365 products and services. For more information, see [Service changes and communication - Microsoft Managed Desktop | Microsoft Docs] (https://docs.microsoft.com/microsoft-365/managed-desktop/service-description/servicechanges?view=o365-worldwide)

Reviewer: ```suggestion
Microsoft Managed Desktop follows a change control process as outlined in our service communication framework. Customers are notified through the Microsoft 365 Message Center and Microsoft Managed Desktop Portal of both security incidents and major changes to the service. Changes to the types of data gathered and where it is stored are considered a material change. We will provide a minimum of 30 days of advanced notification of this change as is standard practice for Microsoft 365 products and services. For more information, see [Service changes and communication](/microsoft-365/managed-desktop/service-description/servicechanges?view=o365-worldwide).
```

## Compliance

> Microsoft Managed Desktop has undergone external audits and obtained a comprehensive set of compliance offerings. You can find more information in [Microsoft Managed Desktop Compliance] (https://docs.microsoft.com/microsoft-365/managed-desktop/intro/compliance). Audit reports are available for download at the Microsoft [Service Trust Portal] (https://aka.ms/stp), which serves as a central repository for Microsoft Enterprise Online Services. (Microsoft Managed Desktop is listed within these documents under the category “Monitoring and Management.”) 

Reviewer: ```suggestion
Microsoft Managed Desktop has undergone external audits and obtained a comprehensive set of compliance offerings. You can find more information in Microsoft Managed Desktop [Compliance](/microsoft-365/managed-desktop/intro/compliance). Audit reports are available for download at the Microsoft [Service Trust Portal](https://aka.ms/stp), which serves as a central repository for Microsoft Enterprise Online Services. (Microsoft Managed Desktop is listed within these documents under the category “Monitoring and Management.”) 
```

## Legal

> Microsoft’s Privacy Notice to End Users of Products Provided by Organizational Customers. Microsoft’s [Privacy Statement](https://privacy.microsoft.com/privacystatement) notifies end users that when they sign into Microsoft products with a work account, a) their organization can control and administer their account (including controlling privacy-related settings), and access and process their data; and b) Microsoft may collect and process the data to provide the service to the organization and end users. 

Reviewer: ```suggestion
**Microsoft’s privacy notice to end users of products provided by organizational customers** - The [Microsoft Privacy Statement](https://privacy.microsoft.com/privacystatement) notifies end users that when they sign in to Microsoft products with a work account, a) their organization can control and administer their account (including controlling privacy-related settings) and access and process their data, and b) Microsoft may collect and process the data to provide the service to the organization and end users.
```
